src/xmipp/libraries/reconstruction/angular_assignment_mag.h

```diff
@@ -0,0 +1,212 @@
+/***************************************************************************
+ *
+ * Authors:     Jeison Méndez García (jmendez@utp.edu.co)
+ *
+ * Instituto de Investigaciones en Matemáticas Aplicadas y en Sistemas -- IIMAS
+ * Universidad Nacional Autónoma de México -UNAM
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef __ANGULAR_ASSIGNMENT_MAG_H
+#define __ANGULAR_ASSIGNMENT_MAG_H
+
+#include "core/xmipp_metadata_program.h"
```

Comment:
Include only necessary files  in the header. The rest should be in the `cpp` file